lib/base/error-handler.js

```diff
@@ -73,7 +73,7 @@ const log = require('npmlog');
             "INVALID_MODE" : "Please specify an option, either '--add' or '--modify'",
             "INVALID_CAPTURE_FORMAT" : "Please specify the file extension(.png, .bmp or .jpg)",
             "INVALID_COMMAND" : "This command is invalid. Please check the supported commands using ares -l",
-            "INVALID_COMBINATION" : "This options is not used with display option",
+            "INVALID_COMBINATION" : "This options can not be used with display option",
```

Comment:
cannot is proper. 
https://m.blog.naver.com/PostView.nhn?blogId=kkwvenus&logNo=221509260030&proxyReferer=https:%2F%2Fwww.google.com%2F